packages/ui/package.json

```diff
@@ -26,9 +26,9 @@
   "dependencies": {
     "@apollo/client": "3.5.7",
     "@hcaptcha/react-hcaptcha": "^1.4.4",
-    "@joystream/js": "1.2.0",
+    "@joystream/js": "^1.6.3",
```

Comment:
It looks good but please reverse the `@joystream/js` update. I'll updated it myself because there is currently a patch which should be removed as well. I think the patch is not necessary anymore but just in case I prefer to remove it in a separate PR which can be easily reversed if something goes wrong.